ogticrd/covid-safe-paths
# Feat: Encrypt data storage phone

#### Description:


Encrypting and Decrypting the data from phone storage
#### Linked issues:


Feat [235](https://trello.com/c/QIMPMnwo/235-as-a-user-i-want-the-data-in-my-phone-storage-encrypted-so-that-data-keeps-safe-even-when-is-saved-in-the-phone-1)

#### How to test:


- `git pull && git checkout Feat/235-encrypt-data-storage-phone`
- `yarn install`
- Run and use the app, all should work as expected
